script/Arcueid/Day 13/13_00_ARC13_4.txt

```diff
@@ -529,8 +529,9 @@ And yet,
 [sha:d79014ed0e0f96e98244f878af98a0d34ed2effc]{
 -- Page 337, Offset 32596.
 -- 「おまえたちは死を拒む事でその強大な力を得るが、同時に弱さをも作り上げた。
-"By rejecting death you people have obtained incredible power,
- but at the same time you have hamstrung yourselves.
+// Too much?
+"By rejecting death, you people have obtained incredible power,
```

Comment:
I'd probably drop the one after `death,` the one after `time` can probably stay

Reply:
I disagree, I like both commas personally.